<!--
    ROS navigation stack with velocity smoother and safety (reactive) controller
-->
<launch>

  <arg name="odom_topic" default="odom" />
  <arg name="laser_topic" default="scan" />

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find neuro_stage_sim)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find neuro_stage_sim)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find neuro_stage_sim)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find neuro_stage_sim)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find neuro_stage_sim)/param/neuro_local_planner_params.yaml" command="load" />
    <!--
    <rosparam file="$(find neuro_stage_sim)/param/dwa_local_planner_params.yaml" command="load" />
    -->
    <rosparam file="$(find neuro_stage_sim)/param/move_base_params.yaml" command="load" />
    <rosparam file="$(find neuro_stage_sim)/param/global_planner_params.yaml" command="load" />
    <rosparam file="$(find neuro_stage_sim)/param/navfn_global_planner_params.yaml" command="load" />

    <remap from="odom" to="$(arg odom_topic)"/>
    <remap from="scan" to="$(arg laser_topic)"/>
  </node>
</launch>
